From a4e877a47adbe2583a39114e63aaa0f690d0f8a3 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Fri, 6 Jul 2012 12:37:54 +0800
Subject: ENGR00215893 [MX6]No need to set high bus when hdmi_clk is enabled

hdmi_clk is used to access hdmi register only, it is not a high
speed clk, no need to set high bus flag.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/mach-mx6/clock.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/mach-mx6/clock.c b/arch/arm/mach-mx6/clock.c
index 79bfd9e0..2173838 100644
--- a/arch/arm/mach-mx6/clock.c
+++ b/arch/arm/mach-mx6/clock.c
@@ -4165,7 +4165,6 @@ static struct clk hdmi_clk[] = {
 	.enable_shift = MXC_CCM_CCGRx_CG2_OFFSET,
 	.enable = _clk_enable,
 	.disable = _clk_disable,
-	.flags = AHB_HIGH_SET_POINT | CPU_FREQ_TRIG_UPDATE,
 	},
 	{
 	 __INIT_CLK_DEBUG(hdmi_iahb_clk)
-- 
1.7.9.5

